### DISCUSSION TITLE
toc-tabbed-internal-nav and toc-tabbed-external-nav behave exactly the same

### DISCUSSION THREAD

**Discussion by Tejeev** [TiddlyWiki/TiddlyWiki5#5296] [Q&A]
There is no discernable difference between the two toc types. Is this a bug or what is the expected behavior? maybe I'm misunderstanding the description and missing it.

**Comment by kookma** [comment]
I would recommend https://groups.google.com/forum/#!forum/tiddlywiki for these questions, they get more attentions and there are more people to reply!

**Reply by dixonge** [reply]
Just out of curiosity, what do you mean by "these questions"? Is there a guideline for what type of questions should be asked here vs. the Google Groups?

**Reply by kookma** [reply]
It seems questions regarding how to use TW (end user mostly) get more attention in Forum.

**Reply by dixonge** [reply]
I guess I would agree, but then why have this area at all?

**Reply by Jermolene** [reply]
Hi @dixonge this area is primarily for discussions about development of JavaScript plugins and the TiddlyWiki core. The Google Group is for end user discussions.

**Reply by dixonge** [reply]
OK, so more of a "place for developers" than a "place to ask a code-related question" or a "place to find out if something is a bug or not".....

got it! :)

**Reply by Tejeev** [reply]
ah, thank you :)
Sorry I missed it in the readme!

**Reply by Tejeev** [reply]
I've asked there.  For the sake of others, I'll come back and post an answer here when I have one
